test/clj/rems/application/test_approver_bot.clj

```diff
@@ -81,13 +78,16 @@
                                 :application/id 1
                                 :application/member {:userid "member"}}
             injections (assoc injections :blacklisted? (fn [user resource]
-                                                         (= ["member" "urn:11"] [user resource])))]
-        (is (empty? (generate-commands [created-event member-added-event submitted-event] injections)))))
+                                                         (= ["member" "urn:11"] [user resource])))
+            application (apply-events [created-event member-added-event submitted-event] injections)]
+        (is (empty? (generate-commands submitted-event injections)))))
```

Comment:
Here the second arg is injections instead of application.

Reply:
done